pkg/core/dao.go

```diff
@@ -0,0 +1,601 @@
+package core
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"github.com/pkg/errors"
+	"sort"
+
+	"github.com/CityOfZion/neo-go/pkg/core/entities"
+	"github.com/CityOfZion/neo-go/pkg/core/storage"
+	"github.com/CityOfZion/neo-go/pkg/core/transaction"
+	"github.com/CityOfZion/neo-go/pkg/crypto/keys"
+	"github.com/CityOfZion/neo-go/pkg/io"
+	"github.com/CityOfZion/neo-go/pkg/util"
+)
+
+// Dao is a data access object.
+type Dao struct {
```

Comment:
It is an abbreviation, wouln't it be better to use all CAPS here?

Reply:
Yes, I think you're right, will rename